ui/coordinateTrainer/src/ctrl.ts

```diff
@@ -49,7 +74,14 @@ export default class CoordinateTrainerCtrl {
   constructor(config: CoordinateTrainerConfig, redraw: Redraw) {
     this.config = config;
     this.colorChoice = (lichess.storage.get('coordinateTrainer.colorChoice') as ColorChoice) || 'random';
+    this.timeControlChoice =
+      (lichess.storage.get('coordinateTrainer.timeControlChoice') as TimeControlChoice) ||
+      document.body.classList.contains('kid')
+        ? 'noTime'
+        : 'thirtySeconds';
+    this.timeDisabled = this.timeControlChoice ? false : true;
```

Comment:
Guessing we want something like this instead:

```suggestion
    this.timeDisabled = this.timeControlChoice  === 'noTime';
```

Reply:
Yes, sorry I'm new to TypeScript.